src/resources/multiple/Review.py

```diff
@@ -0,0 +1,65 @@
+from typing_extensions import ParamSpec
+from bson import ObjectId
+from db.driver import DBdriver
+from flask_restful import Resource, reqparse
+
+class MultipleReview(Resource):
+
+  def __init__(self) -> None:
+    self.db = DBdriver()
+    self.drink_dne = {
+      "data": {
+        "res": None,
+        "err": "Review with the _id DNE"
+      }
+    }, 404
+    self.parser = reqparse.RequestParser(bundle_errors = True)
+
+  def get(self) -> tuple[dict, int]:
+    self.parser.add_argument("_ids", type = str, action = "append")
+    self.parser.add_argument("sample", type = int)
+
+    args = self.parser.parse_args()
+
+    if args["_ids"] is not None and args["sample"] is not None:
+      return {"data": { "err": "Cannot pass both _ids and sample parameters; choose one." } }, 400
+    elif args["_ids"] is not None:
+      if not len(args["_ids"]):
+        return { "data": { "err": "Parameter `_ids` cannot be empty." } }, 400
+    else:
+      sample = 10 if args["sample"] is None else args["sample"]
+      res = [ review.toJSON() for review in self.db.sampleReviews(sample) ]
+
+    return { "data": res }, 200
+
+    def post(self) -> tuple[dict, int]:
+
+      self.parser.add_argument('user_email', type = str)
+      self.parser.add_argument('drink_id', type = str)
+      self.parser.add_argument('comment', type = str, action = "append")
```

Comment:
Any particular reason for having `"comment"` as `action = "append"`? The way I imagine things, comment would just be a single string, but if you have a different/better use case in mind please do share :)

Reply:
I agree with you I think string would make more sense here.